src/ui/public/chrome/directives/kbn_chrome.js

```diff
@@ -26,14 +27,24 @@ export default function (chrome, internals) {
       },
 
       controllerAs: 'chrome',
-      controller($scope, $rootScope, $location, $http) {
+      controller($scope, $rootScope, $location, $http, Private) {
+
+        let unhashStates;
+        try {
+          unhashStates = Private(UnhashStatesProvider);
+        } catch (err) {
+          // we need to work correctly if appState and globalState aren't available
```

Comment:
Actually, there are apps (like console, or status) that don't use appState, and including them would be undesirable, so the chrome needs to behave without them around.